backend/src/handlers/alert.py

```diff
@@ -21,52 +21,36 @@ class AlertModel(BaseModel):
 
 
 @router.get("/")
-def get_alerts(
-    req: Request, include: Union[List[str], None] = Query(default=None)
-) -> JSONResponse:
+def get_alerts(req: Request) -> JSONResponse:
```

Comment:
Okay, so it's fine if we don't include, but please add a query param for active and filter by active/non active alerts